### DISCUSSION TITLE
Headphone Sound inverted only in Hubs - how can I switch it?

### DISCUSSION THREAD

**Discussion by Felixkrell** [mozilla/hubs#4025] [General Questions & Discussion] [answered]
Hey, quickly wanted to ask this:

When I use Hubs on different PCs, in different locations, I get the same problem everytime: My left headphone cup is giving the right signal and vice versa - this only happens when in hubs, not in any other application I use. Others do not seem to have this problem, So I wonder what this could mean.

Is there an option to switch left and right sound in Hubs? That would solve my problems :)

Greetings,

Felix Krell

**Comment by johnshaughnessy** [comment]
This is surprising. Does it happen no matter what avatar you are using? (I could imagine a scenario where an avatar's scale is inverted somehow which might cause the 3D sound to reverse left/right.)

**Comment by pattersonbl2** [comment, marked as answer]
Looks like this issue has become stale. I will mark as answered if more help is needed, please unmark as answered.
